test/suites/machine/machine_test.go

```diff
@@ -91,7 +93,9 @@ var _ = Describe("StandaloneMachine", func() {
 		machine = env.EventuallyExpectCreatedMachineCount("==", 1)[0]
 		Expect(resources.Fits(machine.Spec.Resources.Requests, node.Status.Allocatable))
 		Eventually(func(g Gomega) {
-			g.Expect(machine.StatusConditions().IsHappy()).To(BeTrue())
+			temp := &v1alpha5.Machine{}
```

Comment:
This seems like something that would better be encapsulated by an EventuallyExceptManchineStatusHappy() function

Reply:
Updated